babel.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = api => {
-  if (api) api.cache(false)
```

Comment:
The complexity of this file is back now that babel-jest and TypeScript have been added back. Not really sure there's much we can do about this though. See https://jestjs.io/docs/en/getting-started#using-typescript

Reply:
Yeah – necessary evil now. Never mind!